docs/api/primitives.md

```diff
@@ -0,0 +1,27 @@
+# Primitives
+
+## Address
+
+A string representation of a network address. An address can be a version 4 IP address, a version 6 IP address,
+DNS name or MAC address.
+
+## AddressType
+
+String value of a network address type. Could be only `HOSTNAME`, `IPV4`, `IPV6` or `MAC`.
+
+## DateTime
+
+Pinpoints a moment in time in the format of ISO8601 standard `yyyy-mm-ddThh:mm:ssZ`, where ”yyy” denotes
+year (4 digits), ”mm” denotes month starting from 01, ”dd” denotes day starting from 01, ”T” is the separator
+between date and time part, ”hh” denotes hour in the 24-hour format (00-23), ”MM” denotes minute (00-59),
```

Comment:
if you use MM for the minutes (because mm is for the months), you should change in the format above (because there are 2 mm-s, and 0 MM-s)